jobs/ci-kubernetes-e2e-gke-1.4-latest-cvm-kubectl-skew.env

```diff
@@ -8,3 +8,6 @@ KUBE_GKE_IMAGE_TYPE=container_vm
 ZONE=us-central1-c
 
 KUBEKINS_TIMEOUT=120m
+
+# Enable when testing upgrades from a version < 1.6 to a version >= 1.6
+ENABLE_LEGACY_ABAC=true
```

Comment:
Only GCE. GKE would need the flag that hasn't rolled out to gcloud yet. So, GKE is effectively always ENABLE_LEGACY_ABAC=true right now.